turbinia/config/recipes/triage-linux.yaml

```diff
@@ -0,0 +1,23 @@
+# Triage recipe for quick initial response (specific to Linux).
+
+globals:
+  jobs_allowlist:
+    - CronExtractionJob
+    - CronAnalysisJob
+    - PlasoJob
+    - PsortJob
+    - FileSystemTimelineJob
+
+plaso_base:
+  task: 'PlasoTask'
+
+  file_filter: [
```

Comment:
I'm not sure where this list came from, but maybe we should add `/tmp` as well since it seems like malware gets dropped there fairly frequently?

Reply:
Done, also added a few more file system locations where malware gets frequently dropped.